erizo_controller/erizoClient/src/Events.js

```diff
@@ -49,6 +49,7 @@ const EventDispatcher = () => {
   that.on = that.addEventListener;
   that.off = that.removeEventListener;
   that.emit = that.dispatchEvent;
+  that.unbindEvents = () => { dispatcher.eventListeners = {}; };
```

Comment:
I think that a name like `removeAllListeners()` would be more similar to the rest of methods here. wdyt? 

Could you also move it to line ~35, just below `removeEventListener() {...}` block.